source/includes/tutorial1/_inkling.html.md

```diff
@@ -5,131 +5,96 @@
 
 To connect our simulation to a BRAIN in the Bonsai platform, we will need an Inkling program that describes the problem and how to teach the AI to solve it. Inkling is a programming language specifically designed for artificial intelligence (AI). It abstracts away the vast world of dynamic AI algorithms that require expertise in machine learning and enables more developers and subject matter experts to create AI.
 
-The problem description includes schemas for the states the BRAIN will receive and the actions it will need to send. The description of how to teach the BRAIN includes the reward, as well as any decomposition of the problem into *concepts* and the sequencing of learning using *lessons*. This first tutorial will use a single concept and a single lesson.
+The problem description includes type definitions for the states the BRAIN will receive and the actions it will need to send. The description of how to teach the BRAIN includes the reward, as well as any decomposition of the problem into *concepts* and the sequencing of learning using *lessons*. This first tutorial will use a single concept and a single lesson.
 
-## Schemas
+## Type
 
-A `schema` in Inkling describes a named record and its contained fields. Each field in a schema has a name and a type. A field may also have a [constraint][1] on values that the data described by this field will take.
+A [type][1] in Inkling describes the format and allowed ranges for a data value.
 
 #### Exercises
 
-* Fill in the state schema
-* Fill in the action schema
+* Fill in the state type
+* Fill in the action type
 
-###### Fill in the state schema
+###### Fill in the state type
 
-```inkling--exercise
-schema GameState
-    # EXERCISE: Add a name (and type) for each variable of x and y position.
+```inkling2--exercise
```

Comment:
note: this code is also in the tutorials repo: https://github.com/BonsaiAI/bonsai-tutorials/blob/master/tutorial1/move_a_point.ink
and https://github.com/BonsaiAI/bonsai-tutorials/blob/master/tutorial1/solution/move_a_point.ink